src/visual.ts

```diff
@@ -1381,11 +1381,11 @@ module powerbi.extensibility.visual {
                 // Update each link related with this node
                 node.links.forEach( (link: SankeyDiagramLink) => {
                     // select link svg element by ID generated in link creation as Source-Destination
-                    d3.select(`#linkLabelPaths${(link.source.label.name || "").replace(/\W*/g,"")}-${(link.destination.label.name || "").replace(/\W*/g,"")}`).attr({
+                    d3.select(`#linkLabelPaths${('_' + link.source.label.name || "").replace(/\W*/g, "")}-${('_' + link.destination.label.name || "").replace(/\W*/g,"")}`).attr({
                         // get updated path params based on actual positions of node
                         d: sankeyVisual.getLinkLabelSvgPath(link)
                     });
-                    d3.select(`#${(link.source.label.name || "").replace(/\W*/g,"")}-${(link.destination.label.name || "").replace(/\W*/g,"")}`).attr({
+                    d3.select(`#${('_' + link.source.label.name || "").replace(/\W*/g, "")}-${('_' + link.destination.label.name || "").replace(/\W*/g,"")}`).attr({
```

Comment:
There is ES6 pattern. I think the better way is `#_${(link.soource ..... 
And underline at start is enough